lib/rules/quotes.js

```diff
@@ -328,12 +330,11 @@ module.exports = {
                         description: settings.description
                     },
                     fix(fixer) {
-                        if (isPartOfDirectivePrologue(node)) {
+                        if (node.parent.type === "ExpressionStatement" && !astUtils.isParenthesised(sourceCode, node)) {
```

Comment:
Shall we also check if the `ExpressionStatement` is at the top level of a program or function body?